alembic.rst

```diff
@@ -0,0 +1,84 @@
+Alembic
+=======
+Alembic is a database migration tool for usage with SQLAlchemy Database. The full documentation can be found here: 
+In invenio the alembic environment is already defined in invenio-db
+
+Creating new revisions
+----------------------
+
+When working with an invenio module that already has alembic revisions, after modifying a model we can create a new alembic revision with:
+
+.. code-block:: console 
+
+    $ invenio alembic revision "Added some column." --path alembic_directory_path
+
+A short message describing the changes is required and the path parameter should point to the alembic directory of the module. If the path is not given the new revision will be placed in the invenio_db/alembic directory.
```

Comment:
If the path is not given the new revision will be placed in the invenio_db/alembic directory **which is not the desired behavior**.